app/uk/gov/hmrc/brm/controllers/BirthEventsController.scala

```diff
@@ -56,10 +56,10 @@ trait BirthEventsController extends BRMBaseController {
 
   private def handleInvalidRequest(request : Request[JsValue], errors: Seq[(JsPath, Seq[ValidationError])])(implicit hc : HeaderCarrier) : Future[Result] = {
     countryAuditor.auditCountryInRequest(request.body)
-    info(CLASS_NAME, "post()", s"error parsing request body as [Payload]")
+    info(CLASS_NAME, "post()", s"[handleInvalidRequest] error parsing request body as [Payload]")
 
     val response = ErrorResponses.getErrorResponseByField(errors)
-    info(CLASS_NAME, "post()", s"Response: $response")
+    info(CLASS_NAME, "post()", s"[handleInvalidRequest] Response: $response")
```

Comment:
seems odd having two logs this close together, also not sure if this need's to be logged on it's own looking at the logs